docs/Materials.md.html

```diff
@@ -1093,6 +1097,53 @@
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+### Blending and transparency: refraction
+
+Type
+:    `string`
+
+Value
+:     Any of `none`, `cubemap`, `screenspace`. Defaults to `none`.
+
+Description
+:     Activates refraction when set to anything but `none`. A value of `cubemap` will only use the
```

Comment:
We should also explain that this might override the blending mode (as we discussed on Friday).

Reply:
Yes, I will complete the doc as I implement things.